core/common/src/MapLayerSettings.ts

```diff
@@ -127,9 +136,12 @@ export interface MapLayerProps {
   transparentBackground?: boolean;
   /** Is a base layer.  Defaults to 'false'. */
   isBase?: boolean;
-  /** Access Key for the Layer, like a subscription key or access token */
+  /** Access Key for the Layer, like a subscription key or access token.
+   * TODO This does not belong in the props object. It should never be persisted.

```

Comment:
@mdastous-bentley I changed toJSON to exclude this property, but it really doesn't belong in MapLayerProps at all. Revisit in future PR please?